pkg/apis/vsphere/v1alpha1/vsphereclusterproviderspec_types.go

```diff
@@ -60,37 +60,68 @@ type VsphereClusterProviderSpec struct {
 	//
 	// The default user for CentOS is "centos".
 	// The default user for Ubuntu is "ubuntu".
+	// +optional
 	SSHAuthorizedKeys []string `json:"sshAuthorizedKeys,omitempty"`
 
+	// SSHKeyPair is the SSH key pair for accessing provisioned machines.
+	// This field is set at runtime and should not be assigned by an operator.
+	// Providing access to existing SSH keys is handled by SSHAuthorizedKeys.
```

Comment:
SSHAuthorizedKeys -> GetSSHAuthorizedKeys?

Reply:
Hi @figo,

`SSHAuthorizedKeys` is the field for a list of public SSH keys that can access the remote machines.

`GetSSHAuthorizedKeys` is a function that returns the concatenation of `SSHAuthorizedKeys` and the plain-text, public key from `SSHKeyPair`, if it is set.

Reply:
yup, it is trivial, my question is regards to the comment:
`Providing access to existing SSH keys is handled by SSHAuthorizedKeys` 
is it means by `function GetSSHAuthorizedKeys`?  thanks

Reply:
It's referring to the field `SSHAuthorizedKeys`. It's a Godoc convention to refer to other fields by name.

Reply:
thx